src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -414,6 +414,94 @@ public static void AtomFeedPositiveTest()
             }
         }
 
+        [Fact]
+        public static void RssEntryPositiveTest()
+        {
+            string filePath = @"RssEntry.xml";
+            string serializeFilePath = Path.GetTempFileName();
+
+            try
+            {
+                SyndicationItem feedObjct = null;
+                using (XmlReader reader = XmlReader.Create(filePath))
+                {
+                    Rss20ItemFormatter rss20ItemFormatter = new Rss20ItemFormatter();
+                    rss20ItemFormatter.ReadFrom(reader);
+                    feedObjct = rss20ItemFormatter.Item;
+                    reader.Close();
+                }
+
+                using (XmlWriter writer = XmlWriter.Create(serializeFilePath))
+                {
+                    Rss20ItemFormatter atomformatter = new Rss20ItemFormatter(feedObjct);
+                    atomformatter.WriteTo(writer);
+                    writer.Close();
+                }
+                // compare file filePath and serializeFilePath
+                XmlDiff diff = new XmlDiff()
+                {
+                    Option = XmlDiffOption.IgnoreComments | XmlDiffOption.IgnorePrefix | XmlDiffOption.IgnoreWhitespace | XmlDiffOption.IgnoreChildOrder | XmlDiffOption.IgnoreAttributeOrder
+                };
+                Assert.True(diff.Compare(filePath, serializeFilePath));
```

Comment:
Please add a error message for the assert so that it would be easier for debugging.

I think we should keep the file at serializeFilePath if the assert fail, which would make debugging easier.

Reply:
I have changed the code and print error message in exception message, would you have some suggestion on this modify?